far/farlang.templ.m4

```diff
@@ -4288,7 +4288,7 @@ MEditConfigCursorAtEnd
 "Kurzor je na konci"
 "Pfeile am Ende"
 upd:"Cursor at the end"
-"Kursor na końcu bloku"
+K&ursor na końcu bloku
```

Comment:
I'll fix it this time, but please pay attention to quotes - this line breaks the build.

Reply:
Thanks a lot!